### PR TITLE
New package: irssi-otr-1.0.2

### PR DESCRIPTION
**Pull request by atweiden**

### CODE REVIEW

**Comment by atweiden** [comment]
This seems like it should build cross-platform, but:

```
configure.ac:28: warning: macro 'AM_PATH_GLIB_2_0' not found in library
configure.ac:32: warning: macro 'AM_PATH_LIBGCRYPT' not found in library
```

I'm reticent to make this `only_for_archs=...`, though. It builds on Arch Linux ARM, though not in a cross-compilation setup, so I assume there's something wonky with the build config here.

**Comment by newbluemoon** [comment]
The corresponding .m4 files aren’t found because they’re under `$XBPS_CROSS_BASE`.
You can replace `./bootstrap` with

    autoreconf -fi -I ${XBPS_CROSS_BASE}/usr/share/aclocal

in pre_configure(). Then you’ll need the following configure args to find the irssi headers:

```
--with-irssi-headers=${XBPS_CROSS_BASE}/usr/include/irssi
--oldincludedir=${XBPS_CROSS_BASE}/usr/include
```

And I don’t think exporting all these shell variables is necessary. The required ones should already have been set by xbps-src. Everything is fine here with them removed.

**Comment by atweiden** [comment]
Build works cross-platform now, with much thanks to @newbluemoon.

> Harvest moon drawing near
> Cornfields bristle glowing blue
> Pumpkins howling bright

File: srcpkgs/irssi-otr/template [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Template file for 'irssi-otr'
+pkgname=irssi-otr
+version=1.0.2
+revision=1
+build_style=gnu-configure
+configure_args="--with-gnu-ld
+ --with-irssi-headers=${XBPS_CROSS_BASE}/usr/include/irssi
+ --oldincludedir=${XBPS_CROSS_BASE}/usr/include"
+hostmakedepends="autoconf automake libtool m4 pkg-config"
+makedepends="glib-devel irssi-devel libgcrypt-devel libotr-devel"
```
Comment (maxice8) [inline srcpkgs/irssi-otr/template]: can be libglib-devel

Comment (atweiden) [reply srcpkgs/irssi-otr/template]: https://travis-ci.org/void-linux/void-packages/jobs/456436582

Comment (maxice8) [reply srcpkgs/irssi-otr/template]: i see, carry on then

**Pull request merged**